md/workspaces-and-repositories.md

```diff
@@ -154,6 +155,11 @@ Bonita Studio integrates the default merge tool of EGit. Here are some hints on
 
 Git worklow offers a lot of other features that are not directly integrated in Bonita Studio. You can still use them using the command line interface ([available here](https://git-scm.com/download/)). To retrieve the location of your repository on your filesystem go to <bonita_studio_install_dir>/workspace/<name_of_the_repository>.
 
+#### Troubleshooting
+
+* **My diagram has many validation issues after a clone / a switch branch operation:** Keep in mind that Bonita artifacts (Business Data Model, Organization, Profiles, Applications ...) are not redeployed when you perform a clone or a switch branch operation. So, if you switch from a branch A to a branch B, you may need for example to deploy manually your BDM to ensure that all the business objects specific to the branch B are available.
```

Comment:
Maybe you could list the exhaustive collection of artifacts that would need manual redeployment...